Lib/defconQt/glyphView.py

```diff
@@ -1472,7 +1833,7 @@ def addHorizontalMetrics(self):
             ("Baseline", 0),
             ("x-height", font.info.xHeight),
             ("Cap height", font.info.capHeight),
-            ("Ascender", font.info.ascender),
+            ("Ascender", font.info.ascender)
```

Comment:
I like trailing comma as it spares you two lines of diff when you add an attribute.